app/uk/gov/hmrc/customs/declaration/model/batchFileUploadModels.scala

```diff
@@ -54,15 +54,18 @@ object FileReference {
   implicit val reader = Reads.of[UUID].map(new FileReference(_))
 }
 
+case class CallbackFields(name: String, mimeType: String, checksum: String)
+object CallbackFields {
+  implicit val format = Json.format[CallbackFields]
+}
+
 case class BatchFile(
   reference: FileReference, // can be used as UNIQUE KEY, upscan-initiate
-  name: String, // user request
-  mimeType: String, // upscan-notify
-  checksum: String, // upscan-notify
+  maybeCallbackFields: Option[CallbackFields], // upscan-notify
   location: URL, // upscan-initiate
   sequenceNumber: SequenceNumber, // derived from user request
   size: Int, // assumption - it appears to be mandatory but is ignored
-  documentType: DocumentationType
+  documentType: DocumentationType // user request
```

Comment:
Any reason the type is `DocumentationType` rather than `DocumentType`? I think `Document` is the correct term.